src/components/tables/VDataTable.js

```diff
@@ -223,6 +223,7 @@ export default {
   created () {
     const firstSortable = this.headers.find(h => !('sortable' in h) || h.sortable)
     this.defaultPagination.sortBy = firstSortable ? firstSortable.value : null
+    this.defaultPagination.rowsPerPage = this.rowsPerPageItems[0]
```

Comment:
Only if you provide an empty array to rowsPerPage prop. I'd call that user error :)